packages/ui/.storybook/main.js

```diff
@@ -6,7 +6,7 @@ function resolve(...paths) {
 }
 
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
why did you remove `mdx`? I guess we'll need it

Reply:
Because I don't know yet where `mdx` will be sourced, in which directory.